lib/modules/dosomething/dosomething_user/dosomething_user.theme.inc

```diff
@@ -69,12 +69,17 @@ function dosomething_user_get_register_link($label, array $class = []) {
 function dosomething_user_preprocess_user_profile(&$variables) {
   // Collect User Account data.
   $uid = $variables['elements']['#account']->uid;
-  $user = user_load($uid);
-  $variables['first_name'] = dosomething_user_get_field('field_first_name', $user, 'ucwords');
-  $variables['last_name'] = dosomething_user_get_field('field_last_name', $user, 'ucwords');
+
+  $nsUser = dosomething_northstar_get_user(
+    dosomething_user_get_northstar_id($uid)
+  );
+
+  $variables['first_name'] = $nsUser->first_name;
+  $variables['last_name'] = $nsUser->last_name;
```

Comment:
Will this error if `last_name` is not set? Or is it `null` or something? Just curious since Last Name isn't a required field.

Reply:
Maybe! i will add a check for it. Good flag.